deploy/crds/enterprise.splunk.com_clustermasters_crd.yaml

```diff
@@ -614,6 +614,31 @@ spec:
                       type: array
                   type: object
               type: object
+            appFrameworkRef:
+              description: App Framework configuration. Refers to the config block
+                for App Framework. Through this config, apps can be installed in an
+                Indexer Cluster. The implementatiom is still TBD.
+              properties:
+                featureEnabled:
+                  description: Is Application Framework Feature Enabled. Default value
+                    is False
+                  type: boolean
+                s3Bucket:
+                  description: App Package Remote Store Bucket
+                  type: string
+                s3Endpoint:
+                  description: App Package Remote Store Endpoint
+                  type: string
+                s3PollInterval:
+                  description: App Package Remote Store Polling interval
```

Comment:
We should probably add the units and default value here so users know what they are configuring.

Also I noticed that it looks like the other code is written with seconds in mind, from our earlier discussion did we want to go with minutes instead?